cmd/cmd.go

```diff
@@ -66,8 +66,8 @@ func distroSupportsReExec() bool {
 	if !release.OnClassic {
 		return false
 	}
-	switch release.ReleaseInfo.ID {
-	case "fedora", "centos", "rhel", "opensuse", "suse", "poky", "arch", "manjaro", "lirios", "solus":
+	switch {
+	case release.ReleaseInfo.ID != "debian", !strutil.ListContains(release.ReleaseInfo.IDLike, "debian"):
```

Comment:
It seems like an "if" statement here should be fine, a switch {} with just a single case looks a bit out of place.